opentelemetry-sdk/src/opentelemetry/sdk/logs/__init__.py

```diff
@@ -111,6 +112,48 @@ def force_flush(self, timeout_millis: int = 30000):
         """
 
 
+class OTLPHandler(logging.Handler):
+    """A handler class which writes logging records, in OTLP format, to
+    a network destination or file.
+    """
+
+    def __init__(self, level=logging.NOTSET, log_emitter=None) -> None:
+        super().__init__(level=level)
+        self._log_emitter = log_emitter or get_log_emitter(__name__)
+
+    def _translate(self, record: logging.LogRecord) -> LogRecord:
+        timestamp = int(record.created * 1e9)
```

Comment:
Just for your information, this [PEP](https://www.python.org/dev/peps/pep-0564/) may be relevant here.